fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabBarTabItemTokens.kt

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.AliasTokens
+import com.microsoft.fluentui.theme.token.StateColor
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+open class TabBarTabItemsTokens : TabItemTokens() {
```

Comment:
Can we rename "TabBarTabItemTokens" to "TabBarItemTokens". Since the tabitems are items under tabbar we need not specify tab item again. It would be redundant and confusing

Reply:
removed TabBarTabItemTokens as the TabBarItemTokens defines all tokens.